content/docs/rfcs/61/README.md

```diff
@@ -0,0 +1,373 @@
+- Control nodes provide archives with historical messages **at least** every 30 days.
+- Control nodes receive all community messages.
+- Control nodes are honest.
+- Control nodes know at least one store node from which it can query historical messages.
+
+These assumptions are less than ideal and will be enhanced in future work. This [forum discussion](https://forum.vac.dev/t/status-communities-protocol-and-product-point-of-view/114) provides more details.
+
+# Overview
+
+The following is a high-level overview of the user flow and features this specification describes. For more detailed descriptions, read the dedicated sections in this specification.
+
+## Serving community history archives
+
+Control nodes go through the following (high level) process to provide community members with message histories:
+
+1. Community owner creates a Status community (previously known as [org channels](https://github.com/status-im/specs/pull/151)) which makes its node a Control node.
+2. Community owner enables message archive capabilities (on by default but can be turned off as well - see [UI feature spec](https://github.com/status-im/feature-specs/pull/36)).
+3. A special type of channel to exchange metadata about the archival data is created, this channel is not visible in the user interface.
+4. Community owner invites community members.
+5. Control node receives messages published in channels and stores them into a local database.
+6. After 7 days, the control node exports and compresses last 7 days worth of messages from database and bundles it together with a [message archive index](#waku-message-archive-index) into a torrent, from which it then creates a magnet link ([Magnet URI scheme](https://en.wikipedia.org/wiki/Magnet_URI_scheme), [Extensions for Peers to Send Metadata Files](https://www.bittorrent.org/beps/bep_0009.html)). 
+7. Control node sends the magnet link created in step 6 to community members via special channel created in step 3 through the Waku network.
+8. Every subsequent 7 days, steps 6 and 7 are repeated and the new message archive data is appended to the previously created message archive data.
+
+## Serving archives for missed messages
+
+If the control node goes offline (where "offline" means, the control node's main process is no longer running), it MUST go through the following process:
+
+1. Control node restarts
+2. Control node requests messages from store nodes for the missed time range for all channels in their community
+3. All missed messages are stored into control node's local message database
+4. If 7 or more days have elapsed since the last message history torrent was created, the control node will perform step 6 and 7 of [Serving community history archives](#serving-community-history-archives) for every 7 days worth of messages in the missed time range (e.g. if the node was offline for 30 days, it will create 4 message history archives)
+
```

Comment:
The only thing it can do is try to grab messages from store nodes. If store nodes don't store messages for longer than 30 days, then this is the limit.

So one implicit requirement is that the control node comes online at least once every 30 days (in practice it's much more often though.)